android/src/main/AndroidManifest.xml

```diff
@@ -4,6 +4,8 @@
     <uses-permission android:name="android.permission.INTERNET" />
     <!-- Ranging/monitoring beacons -->
     <uses-permission android:name="android.permission.BLUETOOTH"/>
```

Comment:
Specify `android:maxSdkVersion="30"` for this permission as it is not needed for Android 12+. Examples here:
https://developer.android.com/guide/topics/connectivity/bluetooth/permissions#declare